packages/data-stores/src/launch/selectors.ts

```diff
@@ -27,19 +27,9 @@ export const getSelectedDomain = ( state: State ): DomainSuggestions.DomainSugge
 	state.domain;
 export const getSelectedPlanProductId = ( state: State ): number | undefined => state.planProductId;
 
-/**
- * Returns the product id of the the last paid plan the user had picked.
- * If they revert to a free plan,
- * this is useful if you want to recommend their once-picked paid plan
- *
- * @param state State
- */
-export const getPaidPlanProductId = ( state: State ): number | undefined => {
-	const productId = state.planProductId;
-	const isFree = select( PLANS_STORE ).isPlanProductFree( productId );
-
-	return productId && ! isFree ? state.planProductId : undefined;
-};
+export const isSelectedPlanPaid = ( state: State ): boolean =>
+	Boolean( state.planProductId ) &&
```

Comment:
Bit of a nit, should we have a more explicit check instead of casting to boolean? It would make the code easier to understand (plus,  _techincally_, `planProductId` is a number and it could hold a value of `0`, which would return `false` when cast to Boolean)


```suggestion
	typeof state.planProductId !== 'undefined' &&
```

Reply:
There is no need for the check at all :D